recipes/drug2cell/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "drug2cell" %}
+{% set version = "main" %}
+{% set sha256 = "a6353f989f92f05b36972d42795f7f071245d351bf05027dc7e3e4a074ea5bb8" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/teichlab/drug2cell/archive/main.tar.gz
```

Comment:
Please use a release/tag!
`main` is a moving target. Two separate builds would produce two different outouts.

Reply:
Thanks for your response!
I opened a new issue in the tool's repo to request a release.
https://github.com/Teichlab/drug2cell/issues/7

Reply:
Thanks for your response!
I changed the source URL